stable/pgadmin/values.yaml

```diff
@@ -73,6 +73,7 @@ ingress:
 env:
   email: chart@example.local
   password: SuperSecret
+  enhanced_cookie_protection: 'False'
```

Comment:
Is it required to be quoted? I think it is not needed, but haven't tested it yet.